src/script/entity/Conversation.ts

```diff
@@ -72,7 +71,7 @@ interface UnreadState {
   pings: PingMessage[];
   selfMentions: ContentMessage[];
   selfReplies: ContentMessage[];
-  systemMessages: SystemMessage[];
```

Comment:
humm this is probably a mistake. The `E2EIVerificationMessage` should be a `SystemMessage`. Maybe this is the only change needed here 🤔